src/Application/Actions/Person/Get.php

```diff
@@ -95,6 +97,33 @@ protected function action(Request $request, array $args): Response
                     }
                 }
             }
+
+            if ($includeTipBalances) {
+                // If the pending tips balance was requested (e.g. by the transfer funds form),
+                // check for the sum of all relevant payment intents.
+                $tipPaymentIntentsPending = $this->stripeClient->paymentIntents->all([
+                    'customer' => $person->stripe_customer_id,
+                ]);
+
+                foreach ($tipPaymentIntentsPending->autoPagingIterator() as $paymentIntent) {
+                    $paymentIntentIsPendingDononFundsTip = (
+                        $paymentIntent->status === 'requires_action' &&
+                        $paymentIntent->payment_method_types === ['customer_balance'] &&
+                        $paymentIntent->metadata->campaignName === 'Big Give General Donations'
```

Comment:
I don't think we need to make 'Big Give General Donations' configurable (deploying new PHP code is actually easier than changing config), but I do think it would be worth making it a constant to make it more noticeable as something we might need to edit in future. Maybe a public constant in this class that's referenced in the test so its not duplicated?

Reply:
Agree – when I thought we were going to need to vary it by env and use the ID like in frontend, I'd planned to use an env var. But I think this is standardised and the metadata is already returned "for free" with Payment Intent lists.

Extracted it to a constant for visibility. We can't actually use it twice in a way that's clear however, because the fixture is JSON. I think this was still worth it though, to give us a clear additional place to explain what is going on with those "non-tip tips" in context.